NewForce-Cohort-9/tabloidmvc-stafford-infection-1
# View All Tags #13

**Description**
I have completed the ticket to view all tags. In doing so I have created a tag management link in the navbar, so when the admin selects it, it directs them to the page with all of the tags, where they are able to see the list of all available tags in alphabetical order.

Trello Ticket: https://trello.com/c/WAMiCprG

Type of Change: New Feature

**Testing Instructions**
1. After making sure you have your repo cloned down into a local repository and have the database added, git fetch --a in the master branch to retrieve this branch named abby-viewalltag-13 that I have pushed up. 
2. git checkout abby-viewalltag-13 to switch to my branch and open up Visual Studio. Run the application in a browser window.
3.  If not already logged in, use the credentials, Username:[admin@example.com]
4. You should see the 'Home' page with a navigation bar at the top of the page.
5. Click the option 'Tag Management'. You will be directed to a new page that lists all of the tags in alphabetical order.
6. It should look as such: C#, Cyclopean Terrors, Family, JavaScript.

**Checklist:**

- I have opened abby-viewalltag-13 in my local repository

- I have ran my application in a browser window to see the 'Tag Management' on the navbar

- After logging in the appropriate credentials, [[admin@example.com](mailto:admin@example.com)], and clicking on 'Tag 
   Management' in the navbar, it directs me to the page where you can view all of the available tags in alphabetical order

- I have added test instructions that can be followed to test this code, to view this new feature.
